config/interpolate_funcs.go

```diff
@@ -1657,3 +1663,74 @@ func interpolationFuncAbs() ast.Function {
 		},
 	}
 }
+
+// interpolationFuncRsaDecrypt implements the "rsadecrypt" function that does
+// RSA decryption.
+func interpolationFuncRsaDecrypt() ast.Function {
+	return ast.Function{
+		ArgTypes:   []ast.Type{ast.TypeString, ast.TypeString},
+		ReturnType: ast.TypeString,
+		Callback: func(args []interface{}) (interface{}, error) {
+			s := args[0].(string)
+			key := args[1].(string)
+
+			b, err := base64.StdEncoding.DecodeString(s)
```

Comment:
Assuming base64 for the ciphertext seems reasonable since it'll probably not be valid UTF-8, but in that case I'd expect `rsaencrypt` to be symmetrical and produce a base64 result as output, which it doesn't seem to. Are these asymmetrical for a reason?